cla_public/apps/contact/fields.py

```diff
@@ -76,7 +80,9 @@ def process_formdata(self, valuelist):
 
     @classmethod
     def _format(cls, value):
-        return '{:%Y%m%d}'.format(value)
+        if value and not isinstance(value, basestring):
```

Comment:
I know but a good refactor could be to move ._format into FormattedChoiceField. then add FormattedChoiceField.get_format_string on which looks at self.FORMAT_STRING then nothing is assumed by FormattedChoiceField about types and we can move this very similar looking fn out of DayChoiceField and TimeChoiceField